extensions/ql-vscode/src/remote-queries/view/RepositoriesSearch.tsx

```diff
@@ -10,20 +9,16 @@ interface RepositoriesSearchProps {
 
 const RepositoriesSearch = ({ filterValue, setFilterValue }: RepositoriesSearchProps) => {
   return <>
-    <TextInput
-      block
-      sx={{
-        backgroundColor: 'var(--vscode-editor-background);',
-        color: 'var(--vscode-editor-foreground);',
-        width: 'calc(100% - 14px)',
-      }}
-      leadingVisual={SearchIcon}
-      aria-label="Repository search"
+    <VSCodeTextField
+      placeholder='Filter by repository owner/name'
+      size='150'
+      ariaLabel="Repository search"
       name="repository-search"
-      placeholder="Filter by repository owner/name"
       value={filterValue}
-      onChange={(e: ChangeEvent) => setFilterValue((e.target as HTMLInputElement).value)}
-    />
+      onInput={(e: ChangeEvent) => setFilterValue((e.target as HTMLInputElement).value)}
```

Comment:
For some reason, `onChange` didn't quite do the right thing for this new component, so I tried `onInput`, which worked fine!